src/components/DepositFee/index.tsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react'
+import Tippy from '@tippyjs/react'
+import { isAddress } from 'ethers'
+import { useConnectWallet } from '@web3-onboard/react'
+
+import Button from '../Button'
+import InputText from '../Inputs/InputText'
+import InputRange from '../Inputs/InputRange'
+import InputToggle from '../Inputs/InputToggle'
+
+import * as S from './styles'
+
+type FeesData = {
+  isChecked: boolean
+  feeRate?: string
+  brokerCommision?: number
+  managerShare?: number
+}
+
+interface IDepositFeeProps {
+  feesData?: Record<string, FeesData>
+  disabledNoEvent?: boolean
+  handleClickToggle: (event: React.ChangeEvent<HTMLInputElement>) => void
+  handleRefferalCommission: (event: React.ChangeEvent<HTMLInputElement>) => void
```

Comment:
It is spelled "referral"

```suggestion
  handleReferralCommission: (event: React.ChangeEvent<HTMLInputElement>) => void
```